src/test/java/org/jenkinsci/plugins/nomad/NomadApiTest.java

```diff
@@ -16,7 +16,8 @@
     private NomadSlaveTemplate slaveTemplate = new NomadSlaveTemplate(
             "300", "256", "100",
             null, "remoteFs", "3","1", Node.Mode.NORMAL,
-            "ams", "0", "image", "dc01", "", "", false, "bridge",""
+            "ams", "0", "image", "dc01", "", "", false, "bridge",
+            "", "true", "/mnt:/mnt", "jenkins"
```

Comment:
Should `"true"` not be `true`? I get a compilation error when it tries to convert the `String` to `Boolean` (during test)